Combinatorial_map/doc/Combinatorial_map/Concepts/GenericMap.h

```diff
@@ -917,12 +919,40 @@ If \link GenericMap::are_attributes_automatically_managed `are_attributes_automa
 \sa `is_insertable_cell_1_in_cell_2`
 \sa `insert_cell_0_in_cell_1`
 \sa `insert_cell_0_in_cell_2`
+\sa `insert_cell_1_between_two_cells_2`
 \sa `insert_dangling_cell_1_in_cell_2`
 \sa `insert_cell_2_in_cell_3<InputIterator>`
 \sa `remove_cell<i>`
 */
 Dart_descriptor insert_cell_1_in_cell_2(Dart_descriptor d1, Dart_descriptor d2);
 
+/*!
+Inserts a 1-cell between the 2-cell containing `d1` and the one containing `d2`. Returns `previous(d1)`, a descriptor on one dart belonging to the new 1-cell.
+\pre `is_insertable_cell_1_between_two_cells_2(d1,d2)`.
+
+If \link GenericMap::are_attributes_automatically_managed `are_attributes_automatically_managed()`\endlink`==true`, call \link CellAttribute::On_merge `Attribute_type<i>::type::On_merge`\endlink(<I>a</I>,<I>a'</I>) is called for all enabled i-attributes, for i>=2, with <I>a</I> the original 2-attribute associated with `d` and <I>a'</I> the new 2-attribute created during the operation. If set, the dynamic onmerge function of i-attributes is also called on <I>a</I> and <I>a'</I>.
+
+\cgalAdvancedBegin
+If \link GenericMap::are_attributes_automatically_managed `are_attributes_automatically_managed()`\endlink`==false`, non void attributes are not updated; thus the generic map can be no more valid after this operation.
+\cgalAdvancedEnd
+
+\sa `is_insertable_cell_1_between_two_cells_2`
+\sa `insert_cell_0_in_cell_1`
+\sa `insert_cell_0_in_cell_2`
+\sa `insert_cell_1_in_cell_2`
+\sa `insert_dangling_cell_1_in_cell_2`
+\sa `insert_cell_2_in_cell_3<InputIterator>`
+\sa `remove_cell<i>`
+*/
+Dart_descriptor insert_cell_1_between_two_cells_2(Dart_descriptor d1, Dart_descriptor d2);
+
+/*! Call `insert_cell_1_in_cell_2` if `is_insertable_cell_1_in_cell_2(d1, d2)`, otherwise call `insert_cell_1_between_two_cells_2`.
```

Comment:
I would add a parenthesis pair after the function name.   And if it was only for getting a link.   I would do the same for all the the '\sa`.  If I remember correctly @lrineau finds it correct not to put parenthesis.